### PR TITLE
[docs] Added details about RadiusProfile parameters

### PR DESCRIPTION
**Pull request by rohithasrk**

### CODE REVIEW

File: docs/source/general/enforcing_limits.rst [hunk 1]
```diff
@@ -3,14 +3,30 @@ Enforcing session and bandwidth limits
...
+
+    - Put daily and all time session limits.
+    - Put daily bandwidth limits.
+    - Default profile for the new users being added.
 
 Adding from admin inteface
 --------------------------
 
-At the url `/admin/django_freeradius/radiusprofile/` one can add the profiles for different types of users. And add the user profile mappings at the url `/admin/django_freeradius/radiususerprofile/`. This helps admins configure user limits easily.
+At the url ``/admin/django_freeradius/radiusprofile/`` one can add the profiles for different types of users. And add the user profile mappings at the url ``/admin/django_freeradius/radiususerprofile/``. This helps admins configure user limits easily. There are attributes defined in the RadiusProfile model which help us enforce limits on the users.
```
Comment (2stacks) [inline docs/source/general/enforcing_limits.rst]: Was trying to test this in the master branch.  I don't see a url mapping for /admin/django_freeradius/radiususerprofile/
```
^admin/ auth/group/
^admin/ django_freeradius/radiuscheck/
^admin/ django_freeradius/radiusreply/
^admin/ django_freeradius/radiusaccounting/
^admin/ django_freeradius/nas/
^admin/ django_freeradius/radiususergroup/
^admin/ django_freeradius/radiusgroupreply/
^admin/ django_freeradius/radiusgroupcheck/
^admin/ django_freeradius/radiuspostauth/
^admin/ django_freeradius/radiusbatch/
^admin/ django_freeradius/radiusprofile/
```

Comment (nemesisdesign) [reply docs/source/general/enforcing_limits.rst]: @2stacks it's managed directly in the user page

Comment (rohithasrk) [reply docs/source/general/enforcing_limits.rst]: Hey @2stacks thanks for pointing out. Now it has been changed.

**Comment by rohithasrk** [comment]
@2stacks check out and let me know if your doubts are cleared. We can then merge it.

**Comment by 2stacks** [comment]
So if I understand correctly the Power User profile technically sets a
daily limit of;

48 Hrs and 100GBs

If this is correct I have no issue with it.  For those that don't make use
of session/bandwidth limits I wanted to be sure it was clear how to
configure it.

**Comment by rohithasrk** [comment]
Great. That's what it means. Basically that's our definition of unlimited use.

**Pull request merged**